src/dispatch/incident/service.py

```diff
@@ -263,49 +263,80 @@ def delete(*, db_session, incident_id: int):
     db_session.commit()
 
 
-def calculate_cost(incident_id: int, db_session: SessionLocal, incident_review=False):
-    """Calculates the incident cost."""
-    # we ge the incident
+def get_engagement_multiplier(participant_role: str):
+    """Returns an engagement multiplier for a given incident role."""
+    engagement_mappings = {
+        ParticipantRoleType.incident_commander: 1,
+        ParticipantRoleType.scribe: 0.75,
+        ParticipantRoleType.liaison: 0.75,
+        ParticipantRoleType.participant: 0.5,
+        ParticipantRoleType.reporter: 0.5,
+    }
+
+    return engagement_mappings.get(participant_role, [])
```

Comment:
Should this default to a list? I'm not sure having a default here makes sense.

Reply:
Good catch. No, I'll remove the default.